resultsdbupdater/utils.py

```diff
@@ -406,6 +406,9 @@ def handle_ci_umb(msg):
             'ci_email': msg_body_ci.get('email'),
         }
 
+    # add optional recipients field
+    result_data['recipients'] = msg_body.get('recipients')
```

Comment:
As long as `msg_body` is some kind of dictionary, what would happen if `recipients` is not present in `msg_body`? I'd expect `get('recipients', [])` , to have safe default.

Reply:
it defaults to ```None``` ... But yeah, seems ```[]``` is a more sane default for a list field, fixing :)

Reply:
should be fixed @happz, thanks!